src/electron/menu.ts

```diff
@@ -41,7 +69,7 @@ export function createMenu(): void {
 									designkitPath,
 									PathUtils.join(filePaths[0], 'designkit')
 								);
-								store.openStyleguide(`${filePaths[0]}/designkit`);
+								store.openStyleguide(`${filePaths[0]} / designkit`);
```

Comment:
Are the added spaces here right? As it's a path I'm pretty sure we don't want them here